.gitignore

```diff
@@ -20,6 +20,6 @@ erl_crash.dump
 *.ez
 
 # Ignore package tarball (built via "mix hex.build").
-divo_kafka-*.tar
```

Comment:
?

Reply:
this, I assume was a copy over from Divo_kafka and was overlooked